sagemaker-python-sdk/dgl_lifesci_bap/pytorch_bap.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<p>Can you please point out what data this using to train the model? Or is it only using a pre-trained model...?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Added these statement "In this example, you train an Atomic Convolutional Networks (ACNN) [1] or PotentialNet [2] model using the PDBBind dataset." and "The PDBBind dataset is automatically downloaded."